protected/humhub/libs/Html.php

```diff
@@ -185,6 +185,12 @@ public static function isPjaxEnabled($options)
         return false;
     }
 
+    public static function addTooltip(&$options, $tooltip)
```

Comment:
@buddh4 is the `&` needed or is this a typo?

Reply:
https://www.php.net/manual/en/language.references.pass.php